controllers/disruption_controller.go

```diff
@@ -168,6 +168,10 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, fmt.Errorf("error handling chaos pods termination: %w", err)
 	}
 
+	if err := r.sendWarnings(instance); err != nil {
+		r.log.Errorw("unable to send user warnings", "err", err)
```

Comment:
Ultimately this is optional, so I don't want to stop the reconcile loop if this fails